src/config-as-code/repo-config-service.ts

```diff
@@ -0,0 +1,55 @@
+import YAML from "yaml";
+import GitHubClient from "../github/client/github-client";
+import { getCloudInstallationId } from "../github/client/installation-id";
+import { getLogger } from "../config/logger";
+import { RepoConfig } from "./repo-config";
+import RepoConfigDatabaseModel from "./repo-config-database-model";
+
+const logger = getLogger("services.config-as-code");
+const CONFIG_PATH = "aconfig.yml";
+
+const getRepoConfig = async (githubInstallationId: number, owner: string, repo: string): Promise<string | null> => {
```

Comment:
Maybe rename to `getRepoConfigFromGitHub()` to make it unmistakenly clear where we get it from? :)